source/agora/consensus/EnrollmentManager.d

```diff
@@ -289,8 +290,8 @@ public class EnrollmentManager
 
     ***************************************************************************/
 
-    public bool addEnrollment (const ref Enrollment enroll,
-        const ref PublicKey pubkey, Height block_height, scope UTXOFinder finder)
+    public bool addEnrollment (in Enrollment enroll,
+        in PublicKey pubkey, in Height block_height, scope UTXOFinder finder)
```

Comment:
Doesn't the `finder` need to have the `in` attribute?

Reply:
Nope, delegate usually should just use `scope`